electron/constants.ts

```diff
@@ -15,10 +16,24 @@ function getLegendaryBin(){
   return bin
 }
 
+// Get the "Desktop" directory with multilingual support, fallback to "~/Desktop"
+function getDesktopPath() {
```

Comment:
Instead of using this ou can simple get the desktop folder using:
`app.getPath('desktop')` importing `app` from electron.

Reply:
oh, first time ever using electron, I didn't know this, thanks! I'll update the PR

Reply:
I couldn't use `app.getPath('desktop')` here in the constants.ts file because electron's `app` is undefined during tests and it makes the tests fail, so I used that directly on the games.ts file